dev-packages/application-manager/src/generator/frontend-generator.ts

```diff
@@ -104,12 +104,14 @@ process.env.LC_NUMERIC = 'C';
 const { join } = require('path');
 const { isMaster } = require('cluster');
 const { fork } = require('child_process');
-const { app, BrowserWindow, ipcMain } = require('electron');
+const { app, BrowserWindow, ipcMain, Menu } = require('electron');
+
+const props = require('../../package.json').theia || {};
```

Comment:
I do not really like this. Is there a chance to reuse your work from [here](https://github.com/theia-ide/theia/pull/3445)? Maybe we should generate the application name.